### PR TITLE
language_gabc

### PR DESCRIPTION
**Pull request by jperon**
Another music typesetting language.

This PR should be merged after #234, as it is based on this branch (to avoid merge conflicts on `manifest.json`).

### CODE REVIEW

File: plugins/language_gabc.lua [hunk 1]
```diff
@@ -0,0 +1,19 @@
+-- mod-version:3
+local syntax = require "core.syntax"
+
+syntax.add {
+  name = "GABC",
+  files = { "%.gabc$" },
+  comment = "%%",
+  patterns = {
+    { pattern = "%%.-\n",               type = "comment"  },
```
Comment (Guldoman) [inline plugins/language_gabc.lua:9]:
```suggestion
    { pattern = "%%.*",                 type = "comment"  },
```

File: plugins/language_gabc.lua [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+    { pattern = "^%w+:",                type = "keyword2" },
+    { pattern = "[%*{}]",               type = "operator" },
+    { pattern = "<[^>]*>",              type = "function" },
+    { pattern = "-?%.?%d+",             type = "number"   },
+    { pattern = { "|", "%)" },          type = "keyword2" },
```
Comment (Guldoman) [inline plugins/language_gabc.lua:14]: Is `|` used in this language?

**Review by Guldoman — COMMENTED** [review]
This needs to be rebased.

**Comment by jperon** [comment]
Just squashed and force-pushed. Should be ready if I didn’t forget anything.

**Pull request merged**